### ISSUE TITLE
How to get VMs to populate in UI?

### ISSUE THREAD

**Issue by dalyaidan1** [shapehq/tartelet#101]
### Your Question

When I load Tartlet there are no VMs listed in the UI, but Tart is able to list them with the [same command here](https://github.com/shapehq/tartelet/blob/main/Packages/VirtualMachine/Sources/VirtualMachineData/Tart.swift#L31). Is there another step I missed?

### Any additional context?

Im not sure its a bug, so just posing a question for now. First time using Tart and Tarlet.

Steps:
1. Install Tart via https://github.com/cirruslabs/tart?tab=readme-ov-file#usage
2. Install Tatlet via https://github.com/shapehq/tartelet/wiki/Installing-Tartelet
3. Complete app section of https://github.com/shapehq/tartelet/wiki/Configuring-Tartelet
4. Complete step 1 of VM section of https://github.com/shapehq/tartelet/wiki/Configuring-Tartelet
5. Click refresh on the virtual machine reload button in Tartlet

Here is what the Tartlet app looks like:
![Screenshot 2024-12-04 at 9 31 43 AM](https://github.com/user-attachments/assets/9cd708fc-c8b2-4edf-a6ff-a4aa358b9b4d)
![Screenshot 2024-12-04 at 9 31 52 AM](https://github.com/user-attachments/assets/7ec8da19-5f95-45fa-ad1f-7b4d4134081e)

Here is what Tart lists in my zsh
![tartinfo](https://github.com/user-attachments/assets/6f0eb2d3-3c11-44eb-9479-065006c74156)

System:
MacBook Air
Chip: Apple M2
OS: Sequoia 15.1.1 (24B91)

Tart
Version: 2.21.0

Tartlet
Version: https://github.com/shapehq/tartelet/releases/tag/0.10.2
There are no logs when Export Logs is clicked

Ive also tried restarting my computer and the app several times.

**Comment by kuhnroyal** [comment]
Did you install tart via brew? Worked out of the box for me.

**Comment by dalyaidan1** [comment]
Yes I installed tart via brew, exact same as https://github.com/cirruslabs/tart?tab=readme-ov-file#usage

**Comment by kuhnroyal** [comment]
I have no idea, according to the source, it would throw an error if tart was not found....

**Comment by dalyaidan1** [comment]
We found that the issue is that homebrew is installed at a different path in the system than the candidate listed here:
https://github.com/shapehq/tartelet/blob/e6bcb76c2ddec8052cc8bb8d697c7f4e75a55be8/Packages/VirtualMachine/Sources/VirtualMachineData/Internal/TartLocator.swift#L19

Could you provide an option to customize this candidate array?

**Comment by kuhnroyal** [comment]
But why is there no error 🤔
